src/CMakeLists.txt

```diff
@@ -199,6 +199,14 @@ add_library(parthenon
   parthenon_mpi.hpp
 )
 
+if (BUILD_SHARED_LIBS)
+  add_library(parthenon SHARED "${parthenon_sources}")
```

Comment:
I think cmake handles that automatically: _ If no type is given explicitly the type is STATIC or SHARED based on whether the current value of the variable BUILD_SHARED_LIBS is ON._ (https://cmake.org/cmake/help/latest/command/add_library.html)

Reply:
Yes, I confirmed that this works, thanks